dao/postgres/pg-meta.js

```diff
@@ -72,6 +73,13 @@ class PgMeta {
                         }
                         metadata.version = stepNumber;
                         instance.pgUtils.setMetadata(metadata, callback);
+                        const apis = utils.loadApis();
```

Comment:
This bit is run after *every* SQL migration step. This is why it failed first - you run it before the last step has been applied to the database, and thus the field does not exist!

It must run exactly once, and that's right after the new migration step (2?) has run!

Reply:
Done, but not sure if I like conditionalizing of the execution.

Reply:
I think it's okay (except for the ===)